Makefile

```diff
@@ -51,6 +51,10 @@ venv: requirements.txt requirements-dev.txt
 install-hooks: venv
 	$(BIN)/pre-commit install -f --install-hooks
 
+.PHONY: lint
+lint:
```

Comment:
yeah I had did that at first but it re-installs the hooks every time which I thought was weird even though it doesn't make any significant difference.